chevah_build

```diff
@@ -126,6 +131,10 @@ case $OS in
             export CFLAGS="-m64"
         fi
     ;;
+    hpux*)
+        # libffi and GMP do not compile with the HP compiler, so we use GCC.
```

Comment:
I think it's in the right place as this is where we set up custom values for CC and CXX, eg. for AIX and Solaris above. However, somewhat counter-intuitively, we don't do that for HP-UX and the comment asserts that. Perhaps we should rephrase it to make this more clear? Suggestions welcome…